collector/receiver/otelarrowreceiver/README.md

```diff
@@ -1,89 +1,108 @@
-to `[address]/[metrics_url_path]` for metrics, to `[address]/[logs_url_path]` for logs.
-The default port is `4318`.  When using the `otlphttpexporter` peer to communicate with this component,
-use the `traces_endpoint`,  `metrics_endpoint`, and `logs_endpoint` settings in the `otlphttpexporter` to set the
-proper URL to match the address and URL signal path on the `otlpreceiver`.
+- `endpoint` (default = 0.0.0.0:4317 for grpc protocol, 0.0.0.0:4318 http protocol):
+  host:port to which the receiver is going to receive data. The valid syntax is
+  described at https://github.com/grpc/grpc/blob/master/doc/naming.md.
 
-### CORS (Cross-origin resource sharing)
+Several helper files are leveraged to provide additional capabilities automatically:
 
-The HTTP/JSON endpoint can also optionally configure [CORS][cors] under `cors:`.
-Specify what origins (or wildcard patterns) to allow requests from as
-`allowed_origins`. To allow additional request headers outside of the [default
-safelist][cors-headers], set `allowed_headers`. Browsers can be instructed to
-[cache][cors-max-age] responses to preflight requests by setting `max_age`.
+- [gRPC settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configgrpc/README.md) including CORS
+- [TLS and mTLS settings](https://github.com/open-telemetry/opentelemetry-collector/blob/main/config/configtls/README.md)
 
-[cors]: https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS
-[cors-headers]: https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_request_header
-[cors-max-age]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Max-Age
+Keepalive settings are vital to the operation of OTel-Arrow, because
+longer-lived streams use more memory.  Since every stream of data is
+different, we recommend experimenting to find a good balance between
+memory usage and stream lifetime.
 
-```yaml
+```
 receivers:
-  otlp:
+  otelarrow:
     protocols:
-      http:
-        endpoint: "localhost:4318"
-        cors:
-          allowed_origins:
-            - http://test.com
-            # Origins can have wildcards with *, use * by itself to match any origin.
-            - https://*.example.com
-          allowed_headers:
-            - Example-Header
-          max_age: 7200
+      grpc:
+		keepalive:
+          server_parameters:
+            max_connection_age: 10m
+            max_connection_age_grace: 1m
 ```
 
-[beta]: https://github.com/open-telemetry/opentelemetry-collector#beta
-[contrib]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib
-[core]: https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol
-[stable]: https://github.com/open-telemetry/opentelemetry-collector#stable
+In the example configuration above, OTel-Arrow streams will have reset
+is initiated after 10 minutes and be forcibly shut down one minute
```

Comment:
seems the word "is" at the start of L99 can be removed

s/will have reset is initiated/will have reset initiated

Reply:
gotcha, but ... this whole section needs to be updated with our learnings from https://github.com/grpc/grpc-go/issues/6504. As a TODO, I will continue working on it in this PR.